docs_src/microservices/application/details/ServiceMetrics.md

```diff
@@ -0,0 +1,51 @@
+---
+title: App Services - Service Metrics
+---
+
+# Application Services - Service Metrics
+
+All application services have the following built-in metrics:
+
+- `MessagesReceived` - This is a **counter** metric that counts the number of messages received by the application service. Includes invalid messages.
```

Comment:
can we move this to a table? something like this?

| Metric | Type | Description |
|---|---|---|
| `Messages Received` | counter | counts the number of messages received by the application service. Includes invalid messages |

Reply:
I like it!!